src/widget/form/chatform.cpp

```diff
@@ -797,8 +797,9 @@ void ChatForm::onScreenshotClicked()
 
 void ChatForm::doScreenshot()
 {
-    if (!screenshotGrabber)
-        screenshotGrabber = new ScreenshotGrabber(this);
+    // TODO: Fix, that one ScreenshotGrabber always in memory.
```

Comment:
`FIXME: don't keep ScreenshotGrabber in memory; this was done to fix a crash`

would be more clear